backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/service/SpecificationDataService.kt

```diff
@@ -71,10 +72,18 @@ class SpecificationDataService @Autowired constructor(
             null
         }
 
-    fun extractId(json: String): String? =
-        try {
-            JsonPath.read<String>(json, "$.info.x-api-id")
+    fun extractId(json: String): String? {
+        val idPath = "$.info.x-api-id"
+        return try {
+            JsonPath.read<String>(json, idPath)
```

Comment:
Do you think we can do it with just Jackson, with `JsonNode.at()` and `JsonNode.asText()`? I think we can avoid the nested `try` with that and also remove another Json dependency ;)